gcloud/bigtable/_helpers.py

```diff
@@ -85,8 +90,11 @@ def make_stub(client, stub_factory, host, port):
     :rtype: :class:`grpc.beta._stub._AutoIntermediary`
     :returns: The stub object used to make gRPC requests to a given API.
     """
+    root_certificates = get_certs()
+    client_credentials = implementations.ssl_client_credentials(
+        root_certificates, private_key=None, certificate_chain=None)
+    channel = implementations.secure_channel(
+        host, port, client_credentials)
     custom_metadata_transformer = MetadataTransformer(client)
-    return stub_factory(host, port,
-                        metadata_transformer=custom_metadata_transformer,
-                        secure=True,
-                        root_certificates=get_certs())
+    return stub_factory(channel, host=host,
```

Comment:
Yes, you saw that right. I don't know why the redundancy is there. Maybe the second `host` is optional and I don't need to be using it?